sdk/search/azure-search-documents/README.md

```diff
@@ -69,7 +69,7 @@ Note that you will need an admin key to index documents (query keys only work fo
 
 <!-- embedme ./src/samples/java/com/azure/search/documents/ReadmeSamples.java#L54-L58 -->
 ```Java
-SearchIndexClient searchIndexClient = new SearchIndexClientBuilder()
+SearchIndexClient searchClient = new SearchIndexClientBuilder()
```

Comment:
I'm not sure I see the point of renaming the local variable if the class itself wasn't renamed to `SearchClient` (the new intended class name).

Reply:
The IDE takes this action when refector.
Will make the readme right.

Reply:
You can check it to only rename code instances.